src/content/structured/get-started/react.mdx

```diff
@@ -27,11 +27,11 @@ In the root of your project:
 
 ```shell
 # using npm
-npm install @ukic/react@next @ukic/fonts@next @ukic/web-components@next
+npm install @ukic/react @ukic/fonts @ukic/web-components
```

Comment:
hopefully once we no longer have the alpha tag on the packages, you should no longer need to install the web-components package for react based frameworks, so we can go back to taking that part out, like we did in v2